homeassistant/components/ipma/weather.py

```diff
@@ -68,71 +67,82 @@
         _LOGGER.error("Latitude or longitude not set in Home Assistant config")
         return
 
-    station = await async_get_station(hass, latitude, longitude)
+    api = await async_get_api(hass)
```

Comment:
Note, this shouldn't happen in this PR. But I think we should do it now in a follow up PR.

Ie, move the config section under the integration domain and just use the config entries api.

Reply:
No, we can import the config yaml to a config entry.

Normally we prefer to have the config section under the integration domain instead of the platform domain, since it is more future proof. Moving the config section will be a breaking change.

If we think there will never be more platforms, the platform can also import the config yaml as a config entry from `async_setup_platform`. This will then not be a breaking change.

Reply:
@MartinHjelmare IPMA is the portuguese weather institute (Portugueses Institute of Sea and Atmosphere to be precise), so I doubt there will be other platforms under the ipma integration.
I'll do that as a new PR then :)